.github/workflows/ci.yml

```diff
@@ -23,9 +23,11 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
-      - name: Install dependencies
-        run: |
-          gem install bundler
-          bundle install --jobs 4 --retry 3
-      - name: Run the default task
-        run: bundle exec rake
+      - name: Lint  files
+        run: bundle exec rake rubocop
+      - name: Run tests
+        run: bundle exec rake spec
```

Comment:
Is this same as - `bundle exec rspec spec`?

Reply:
Yes!

https://github.com/solidusio/rubocop-solidus/blob/c62eb78b1e5bbb77b46f2f91ed64c6e493cdb78f/Rakefile#L13-L15